docs/content/en/install/eks/_index.md

```diff
@@ -389,7 +389,33 @@ Take note of the comments in this file. If you have changed the name of your sec
 
 ### Deploy K8ssandra with Helm
 
-With a `values.yaml` file generated which details out specific configuration overrides we can now deploy K8ssandra via Helm.
+If you haven't already, add the latest stable K8ssandra repo:
+
+```bash
+helm repo add k8ssandra https://helm.k8ssandra.io/stable
```

Comment:
You can drop the `stable`

Reply:
Thanks Chris - updated in recent commit.